dev/tuple_helper/tuple_iteration.h

```diff
@@ -33,7 +33,8 @@ namespace sqlite_orm {
         template<bool reversed = false, class Tpl, size_t... Idx, class L>
         void iterate_tuple(const Tpl& tpl, std::index_sequence<Idx...>, L&& lambda) {
             if constexpr(reversed) {
-                iterate_tuple(tpl, reverse_index_sequence(std::index_sequence<Idx...>{}), std::forward<L>(lambda));
+                int sink;
```

Comment:
Please put a comment above similar to:
`// nifty fold expression trick: make use of guaranteed right-to-left evaluation order when folding over operator=`.

Even seasoned C++ programmers will find it helpful :) - Even I had to try it out and find out about it.